netcdf4/src/test/groovy/ucar/nc2/jni/netcdf/Nc4IospSpec.groovy

```diff
@@ -1,46 +0,0 @@
-package ucar.nc2.jni.netcdf
-
-import org.junit.Assume
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
-import spock.lang.Specification
-import ucar.nc2.Attribute
-
-/**
- * Test various aspects of Nc4Iosp.
- *
- * @author cwardgar
- * @since 2016-12-27
- */
-class Nc4IospSpec extends Specification {
-    private static final Logger logger = LoggerFactory.getLogger(Nc4IospSpec)
-    
-    def setup() {
-        // Ignore this class's tests if NetCDF-4 isn't present.
-        // We're using setup() because it shows these tests as being ignored.
-        // setupSpec() shows them as *non-existent*, which is not what we want.
-        Assume.assumeTrue("NetCDF-4 C library not present.", Nc4Iosp.isClibraryPresent());
```

Comment:
I think we still need to check that the C library is present for these tests. You can follow what this test does: https://github.com/Unidata/tds/blob/13017bd6dfd9f5ae270bf2e524d8d7340592742b/tds/src/test/java/thredds/server/ncss/view/dsg/DsgSubsetWriterTest.java#L47

Reply:
If I understand the comment correctly they wanted the tests to be ignored if the C Library is present. But these tests just test that no exception is thrown when a method is called, which should be the case whether or not the library exists, so I don't think that assumption is really needed

Reply:
wait, what are these even meant too be testing?

Reply:
My understanding was that at some point if you didn't have the netcdf-c library present then calling these functions caused a null pointer exception. Then they made these tests and added a null check to fix that. So they only check that there is no null pointer exception thrown, but yeah not the most useful tests of all time 😆

Reply:
lol well now they don't do anything cuz the functions return right away 🤷

Reply:
yeah it's not a super useful test... do you still want to check for the c library or leave as is?

Reply:
We ended up adding the check for the C library